runtime/r_wrap.c

```diff
@@ -180,14 +184,17 @@ void processCmd(char *args[])
             }
             close(inputFd);
 
-            //write block to stdout
+            // write block to stdout
             writeHeader(stdout, id, currReadLen);
             safeWriteWithFlush(stdoutBlock, 1, currReadLen, stdout);
 
-            //update header (ordered at the end so !feof works) and cleanup
+            // update header (ordered at the end so !feof works) and cleanup
             readHeader(stdin, &id, &blockSize);
+
+            // wait is necessary to reclaim process
+            // Do I need to do something with return status?
```

Comment:
I think the return status of the parallel does not correspond to the one from the sequential so it can be ignored.